packages/devtools_app/lib/src/shared/analytics/analytics_controller.dart

```diff
@@ -49,14 +49,13 @@ class AnalyticsController {
 
   /// Method to call to confirm with package:unified_analytics the user has
   /// seen the consent message.
-  final AsyncAnalyticsCallback? _markConsentMessageAsShown;
   Future<void> markConsentMessageAsShown() async =>
-      await _markConsentMessageAsShown?.call();
+      await dtdManager.analyticsConsentMessage();
```

Comment:
Is the right function being called here?

```suggestion
      await dtdManager.analyticsClientShowedMessage();
```